public/src/pages/main/Header/Nav.tsx

```diff
@@ -93,8 +93,8 @@ export const Nav = () => {
   return (
     <nav className="nav-menu d-none d-lg-block">
       <ul>
-        {menu.map((item, i) =>
-          !item.admin || isAdminUser ? menuItem(item, i) : <></>
+        {menu.map(
+          (item, i) => (!item.admin || isAdminUser) && menuItem(item, i)
```

Comment:
Please revert the changes, the preferred way is to use the ternary operator

Reply:
@derevnjuk done